yellow-paper/CallsAndContexts.mdx

```diff
@@ -0,0 +1,13 @@
+# Calls and Contexts
```

Comment:
In this pr?

Reply:
Graphite added some files I didn't mean to add